pom.xml

```diff
@@ -177,6 +177,25 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
         <version>2.3.1</version>
+        <configuration>
+          <archive>
+            <manifest>
+              <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+              <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+            </manifest>
+            <manifestEntries>
+              <Bundle-Version>${parsedVersion.osgiVersion}</Bundle-Version>
+              <Bundle-Description>${project.description}</Bundle-Description>
+              <Os-Name>${os.name}</Os-Name>
+              <Os-Arch>${os.arch}</Os-Arch>
+              <Os-Version>${os.version}</Os-Version>
+              <X-Compile-Source-JDK>${maven.compiler.source}</X-Compile-Source-JDK>
```

Comment:
Is it part of any standard? There is [Bundle-RequiredExecutionEnvironment](http://wiki.osgi.org/wiki/Bundle-RequiredExecutionEnvironment) which is part of OSGi. If you decide to change to it, make sure you use correct string from http://wiki.osgi.org/wiki/Execution_Environment.